setuptools/dist.py

```diff
@@ -228,6 +238,9 @@ def write_field(key, value):
         for extra in self.provides_extras:
             write_field('Provides-Extra', extra)
 
+    if version >= StrictVersion('2.1'):
```

Comment:
You use this comparison twice in this patch. It's time to put it into a var, don't you think? You could have nicer checks like `if at_least_2_1:` and `if not at_least_2_1:`

Reply:
I don't really think that it's necessary. At least not just yet. There are also two checks for version `1.2` without an extra variable.

Reply:
Fair enough